.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build
```

Comment:
A comment before this file would be helpful, perhaps something like:
`# CI testing for building patroni docker image`

Also not sure if its required but you might want the:
```
# Copyright 2021 Canonical Ltd.
# See LICENSE file for licensing details.
```

Reply:
Thanks Mia. Same as the comment above.